www/templates/apl-config.html

```diff
@@ -16,7 +16,7 @@
  * You should have received a copy of the GNU General Public License
  * along with this program.  If not, see <http://www.gnu.org/licenses/>.
  *
- * 2019-05-07 TC moOde 5.2
+ * @@__CURRENTDATE__ TC moOde @@__VERSION__
```

Comment:
Changed one source to demonstrated use central setting for auto replacing of current date and version in deployed files.